runtime/doc/nvim.txt

```diff
@@ -16,11 +16,14 @@ differences from Vim.
 ==============================================================================
 Transitioning from Vim				*nvim-from-vim*
 
-To start the transition, link your previous configuration so Nvim can use it:
+To start the transition, create the `init.vim` file and make it source your
+existing configuration:
 >
-    mkdir ~/.config
-    ln -s ~/.vim ~/.config/nvim
-    ln -s ~/.vimrc ~/.config/nvim/init.vim
+    mkdir -p ~/.config/nvim
+    touch ~/.config/nvim/init.vim
+    echo 'set runtimepath+=~/.vim,~/.vim/after' >> ~/.config/nvim/init.vim
+    echo 'set packpath+=~/.vim' >> ~/.config/nvim/init.vim
+    echo 'source ~/.vimrc' >> ~/.config/nvim/init.vim
```

Comment:
The shell steps look more noisy/less convenient, even though the new approach is intended to actually reduce room for error/confusion. how about:

```
To start the transition, create ~/.config/nvim/init.vim with these contents: >
    set runtimepath+=~/.vim,~/.vim/after
    set packpath+=~/.vim
    source ~/.vimrc
<
```